wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.15.0/2__add_commands_to_k8s_runtime.sql

```diff
@@ -0,0 +1,41 @@
+--
+-- Copyright (c) 2012-2018 Red Hat, Inc.
+-- This program and the accompanying materials are made
+-- available under the terms of the Eclipse Public License 2.0
+-- which is available at https://www.eclipse.org/legal/epl-2.0/
+--
+-- SPDX-License-Identifier: EPL-2.0
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+
+-- Commands --------------------------------------------------------------------
+CREATE TABLE k8s_runtime_command (
+    id              BIGINT         NOT NULL,
+    commandline     TEXT,
+    name            VARCHAR(255)   NOT NULL,
+    type            VARCHAR(255)   NOT NULL,
+    workspace_id    VARCHAR(255),
```

Comment:
Is it possible that WS ID is null here?

Reply:
copied/pasted from `1__init.sql`#command for WorkspaceConfig, I thought it was written in this way when schema was generated by JPA. 
Checked: `workspace_id` must be nullable because in this case EclipseLink work in the following way: insert entity that is not related to a workspace, update workspace id in inserted object
```
[EL Fine]: sql: 2018-11-26 11:51:22.071--ClientSession(945064729)--Connection(678962690)--Thread(Thread[main,5,main])--INSERT INTO k8s_runtime_command (id, commandline, name, type) VALUES (?, ?, ?, ?)
	bind => [6, echo command9fz9e, commandutbry, CUSTOM]
[EL Fine]: sql: 2018-11-26 11:51:22.071--ClientSession(945064729)--Connection(678962690)--Thread(Thread[main,5,main])--UPDATE k8s_runtime_command SET workspace_id = ? WHERE (id = ?)
	bind => [wsId0l8yy6ut, 6]
```
I'm not aware whether we can configure this behavior. I would leave it nullable since it is not the first place where it is done in this way.

Reply:
ok